integration-cli/docker_api_info_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+)
+
+func TestInfoApi(t *testing.T) {
```

Comment:
Can we have simple test for `cli` too? Because it was changed. Or maybe there is already tests?